node/bundle_error.ts

```diff
@@ -30,6 +32,11 @@ class BundleError extends Error {
  */
 const wrapBundleError = (input: unknown, options?: BundleErrorOptions) => {
   if (input instanceof Error) {
+    if (input.message.includes("The module's source code could not be parsed")) {
```

Comment:
Same here, if you don't mind.

Reply:
I don't think that works - we want to wrap it in a `BundleError`, even if the message is something different

Reply:
Oops, sorry. Didn't look at this properly.